modules/react-native-desktop-shortcuts/desktop/desktopshortcuts.cpp

```diff
@@ -0,0 +1,81 @@
+#include "desktopshortcuts.h"
+#include "bridge.h"
+
+#include "eventdispatcher.h"
+#include <QDebug>
+#include <QCoreApplication>
+#include <QKeySequence>
+#include <QEvent>
+#include <QKeyEvent>
+
+Q_LOGGING_CATEGORY(DESKTOPSHORTCUTS, "DesktopShortcuts")
+
+namespace {
+struct RegisterQMLMetaType {
+  RegisterQMLMetaType() { qRegisterMetaType<DesktopShortcuts *>(); }
+} registerMetaType;
+} // namespace
+
+DesktopShortcuts::DesktopShortcuts(QObject *parent)
+    : QObject(parent) {
+  QCoreApplication::instance()->installEventFilter(this);
+  connect(this, &DesktopShortcuts::shortcutInvoked, this, &DesktopShortcuts::onShortcutInvoked);
+}
+
+DesktopShortcuts::~DesktopShortcuts() {
+}
+
+void DesktopShortcuts::setBridge(Bridge *bridge) {
+  this->bridge = bridge;
+}
+
+QString DesktopShortcuts::moduleName() { return "DesktopShortcutsManager"; }
+
+QList<ModuleMethod *> DesktopShortcuts::methodsToExport() {
+  return QList<ModuleMethod *>{};
+}
+
+QVariantMap DesktopShortcuts::constantsToExport() { return QVariantMap(); }
+
+void DesktopShortcuts::registerShortcuts(const QStringList& shortcuts) {
+  //qCDebug(DESKTOPSHORTCUTS) << "registerShortcuts" << shortcuts;
+  this->registeredShortcuts = shortcuts;
+}
+
+bool DesktopShortcuts::eventFilter(QObject* obj, QEvent* event) {
+  if (event->type() == QEvent::KeyPress) {
+    QKeyEvent* ke = static_cast<QKeyEvent*>(event);
+
+    QString modifier;
+
+    if (ke->modifiers() & Qt::ShiftModifier) {
+      modifier += "Shift+";
+    }
+    else if (ke->modifiers() & Qt::ControlModifier) {
```

Comment:
There can be more modifiers than one. So we can use `if` instead of `else if` everywhere :)

Reply:
Oops, thanks @vkjr!